.github/workflows/coverage-linux-without-intl.yml

```
@@ -32,43 +38,46 @@ concurrency:
+  CXX: sccache clang++
+  SCCACHE_GHA_ENABLED: 'true'
 
 permissions:
   contents: read
 
 jobs:
   coverage-linux-without-intl:
-    if: github.event.pull_request.draft == false
-    runs-on: ubuntu-latest
+    # Disabled because "Report JS" step was crashing.
+    if: false
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)